### PR TITLE
feat: change alert title to be more consistent with figma designs

### PR DESCRIPTION
**Pull request by georgesimos**
### Background

Reduce the font size by one level on Alert title 

Figma Designs : 

- https://www.figma.com/file/jaP87DKwYCoO3o1bAYp6OH/02-System-Health-%E2%9C%85?node-id=81%3A7579
- https://www.figma.com/file/LkZk4NngNuOGFH3VNzS36z/08-Integrations-%E2%9C%85?node-id=851%3A38751

### Changes

- Alert component

### Testing

- Manual

#### Before
Screenshot 2021-12-02 at 12 29 49 PM

#### After 
Screenshot 2021-12-02 at 12 33 31 PM

### CODE REVIEW

**Comment by 3nvi** [comment]
Can you quickly help me by aadding:

- before & after screenshots
- link to the figma designs


Thanks!

File: src/components/utils/ControlledAlert/ControlledAlert.tsx [hunk 1]
```diff
@@ -74,13 +74,13 @@ const ControlledAlert = React.forwardRef<HTMLDivElement, ControlledAlertProps>(
         >
           <Box>
             {title && (
-              <Flex as="header" align="center" fontSize="large">
+              <Flex as="header" align="center">
                 {icon && <Icon type={icon} mr={2} color={iconColor} size="large" />}
                 <Box
-                  as="h4"
+                  as="h2"
```
Comment (3nvi) [inline src/components/utils/ControlledAlert/ControlledAlert.tsx:80]: I'm not sure if `h2` is the right call, but I'm not sure about `h4` either. Wouldn't want to block it on that, so your call :)

**Review by 3nvi — APPROVED** [review]
LGTM 🎉

File: src/components/utils/ControlledAlert/ControlledAlert.tsx (hunk 1 above)
Comment (georgesimos) [reply src/components/utils/ControlledAlert/ControlledAlert.tsx:80]: I wanted to handle alert as individual element, so I used the h2 to specify the importance of the title!

**Pull request merged**